controllers/subscriptions.go

```diff
@@ -81,6 +83,19 @@ func SetBundleInfo(yamlFilePath string) error {
 	return nil
 }
 
+func setSubscriptionsQueryFeatures() {
+	features := strings.Split(configOptions.GetString(config.Keys.Features), ",")
+
+	var skuBasedFeatures []string
+	for _, bundle := range bundleInfo {
+		if slices.Contains(features, bundle.Name) && bundle.Skus != nil && len(bundle.Skus) > 0 {
```

Comment:
looking at this again, I probably should have put the `slices.Contains` bit at the end of this conditional. That way we save ourselves from running a nested loop unless we actually need to... but I don't actually know if go executes every part of the conditional at once or serially? Regardless this method will only run once per app startup so its not like we are scaling this code with traffic

Reply:
ok as expected go does not evaluate the entire conditional if one part of it returns false: https://go.dev/ref/spec#Logical_operators